test/routers/fixtures.py

```diff
@@ -1,17 +1,18 @@
+from laa_court_data_api_app.models.token_response import TokenResponse
+from laa_court_data_api_app.models.prosecution_cases.prosecution_cases_results import ProsecutionCasesResults
+from laa_court_data_api_app.models.prosecution_cases.prosecution_cases import ProsecutionCases
+from laa_court_data_api_app.models.hearing_summaries.hearing_summary import HearingSummary
+from laa_court_data_api_app.models.prosecution_cases.defendant_summary import DefendantSummary
+from laa_court_data_api_app.models.hearing_events.hearing_events_result import HearingEventsResult
+from laa_court_data_api_app.models.hearing.hearing_result import HearingResult
+from laa_court_data_api_app.models.hearing.hearing import Hearing
 import datetime as dt
```

Comment:
The IDEs seem to do this automatically anyway and I think it helps to organise the imports to be grouped by internal imports (so anything from our app directory) in one block, then imports of libraries/modules in another

Reply:
could i get some clarification here please. I'm not sure if this is a change suggestion or informative

Reply:
Sorry I somehow managed to remove some part of my comment - I meant that it would be nice to group our imports so that internals and libraries are separate